netci.groovy

```diff
@@ -230,10 +231,13 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
             baseName = architecture.toLowerCase() + '_cross_' + configuration.toLowerCase() + '_' + os.toLowerCase()
             break
         case 'x86ryujit':
-            baseName = architecture.toLowerCase() + configuration.toLowerCase() + '_' + os.toLowerCase()
+            def arch = 'x86'
```

Comment:
I agree with adiaaia.  Why so many variables?  Seems like you might as well hardcode arch and jit inside the strings below.